src/security_hub_reporter.py

```diff
@@ -0,0 +1,130 @@
+import boto3, os, json
+
+sns = boto3.client('sns')
+securityhub = boto3.client('securityhub')
+cloudwatch = boto3.client('cloudwatch')
+
+GENERATOR_ID = 'aws-foundational-security-best-practices/v/1.0.0'
+
+SECURITY_CONTROLS = os.environ.get('SECURITY_CONTROLS').replace(' ', '').split(',')
```

Comment:
I am not a python expert but how about this:
`os.environ.get('SECURITY_CONTROLS').strip().split(',')`